server/scripts/start-test-server.sh

```diff
@@ -87,6 +87,13 @@ for (( i=1; i<=$max_retries; i++ )); do
 
   if check_containers_health; then
     echo "All containers are healthy!"
+    CREATED_SERVICES=$(docker compose -p $APPLICATION ps -a --services)
+    for service in $CREATED_SERVICES; do
+      echo "Copy test sdk to $service"
+      docker compose -p $APPLICATION cp "${DIR}/../test-data/sdk/a" "$service:/var/extender/sdk"
+      docker compose -p $APPLICATION exec -u root $service chown -R extender:extender /var/extender/sdk/a
```

Comment:
Change folder ownership because after `cp` command it has ownership of user `dialout`. That leads to fails during tests execution because of `access denied`.